packages/front-end/components/Features/ScheduleInputs.tsx

```diff
@@ -0,0 +1,142 @@
+import { ScheduleRule } from "back-end/types/feature";
+import React, { useEffect, useState } from "react";
+import Field from "../Forms/Field";
+import SelectField from "../Forms/SelectField";
+import { GBAddCircle } from "../Icons";
+
+interface Props {
+  defaultValue: ScheduleRule[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onChange: (value: any) => void;
+}
+
+const getLocalDateTime = (rawDateTime: string) => {
+  if (!rawDateTime) {
+    return "";
+  }
+  const utcDateTime = new Date(rawDateTime);
+
+  // We need to adjust for timezone/daylight savings time before converting to ISO String to pass into datetime-local field
+  utcDateTime.setHours(
+    utcDateTime.getHours() - new Date(rawDateTime).getTimezoneOffset() / 60
+  );
+  return utcDateTime.toISOString().substring(0, 16);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export default function ScheduleInputs(props: Props) {
+  const [rules, setRules] = useState(props.defaultValue);
+
+  useEffect(() => {
+    props.onChange(rules);
+  }, [props, props.defaultValue, rules]);
+
+  const options = [
+    { label: "enabled", value: "true" },
```

Comment:
Having to set the value here to a string is a bit weird. Is there a better way to do this?